### ISSUE TITLE
Fix a11y-Issue in pFocusTrap

### ISSUE THREAD

**Issue by geromegrignon** [openng-org/optimus-ui#418]
### Describe the bug

if you use pFocusTrap, there is an a11y-violation.
Following is the error message from axe:
```
Violation found at $('span[data-pc-section="lastfocusableelement"]')
<span class="p-hidden-accessible p-hidden-focusable" tabindex="0" role="presentation" aria-hidden="true" data-p-hidden-accessible="true" data-p-hidden-focusable="true" data-pc-section="lastfocusableelement"></span>
Received:
ARIA hidden element must not be focusable or contain focusable elements (aria-hidden-focus)
Fix all of the following:
  Focusable content should have tabindex="-1" or be removed from the DOM
You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.10/aria-hidden-focus?application=axeAPI
────────
```

so there is an element, that is hidden (aria-hidden="true") and tabbable (tabindex="0").
Both together is a violation.

### Reason for not contributing a PR

- [x] Lack of time
- [x] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://primeng.org/focustrap#basic

### Environment

All Environments

### Angular version

19.2.9

### PrimeNG version

v19

### Node version

20.15.0

### Steps to reproduce the behavior

1. Go To https://primeng.org/focustrap#basic
2. open console
3. type `$('span[data-pc-section="lastfocusableelement"]')`
4. inspect found element
5. see `aria-hidden="true"` with `tabindex="0"`

### Expected behavior

There should not be a hidden Element that is tabbable.

> Migrated from `primefaces/primeng#18495` — originally by `@B34v0n` on 2025-06-18

**Comment by geromegrignon** [comment]
*[Originally posted by `@jase88` on 2025-06-18T12:49:22.000Z]*

Violation against the [4th aria rule](https://www.w3.org/TR/using-aria/#4thrule)

> Do not use _role="presentation"_ or _aria-hidden="true"_ on a __focusable element__.

**Comment by geromegrignon** [comment]
*[Originally posted by `@github-actions[bot]` on 2025-12-16T00:21:43.000Z]*

This issue has been automatically marked as stale due to a lack of activity. **If this issue is still relevant to you, please leave a comment** so we can keep it open. We apologize for not being able to prioritize it sooner. If you have any new information or questions, please share them in your comment!

**Comment by geromegrignon** [comment]
*[Originally posted by `@B34v0n` on 2025-12-18T13:41:23.000Z]*

This issue is still relevant
